notebook/nbextensions.py

```diff
@@ -165,17 +142,19 @@ def install_nbextension(path, overwrite=False, symlink=False, user=False, prefix
             filename = urlparse(path).path.split('/')[-1]
             local_path = os.path.join(td, filename)
             if verbose >= 1:
-                print("downloading %s to %s" % (path, local_path))
+                log("Downloading: %s -> %s" % (path, local_path))
             urlretrieve(path, local_path)
             # now install from the local copy
-            install_nbextension(local_path, overwrite=overwrite, symlink=symlink, nbextensions_dir=nbext, destination=destination, verbose=verbose)
+            nsion(local_path, overwrite=overwrite, symlink=symlink,
```

Comment:
This is an awesome feature of the Atom code editor!  It's called "introduce bugs in random text files"  :wink: